src/StorageWriter/StorageWriter.ts

```diff
@@ -42,21 +56,33 @@ export class StorageWriter {
     this.router = this.container.get('Router')
     await this.router.start()
 
-    await this.createIndices()
+    this.service = this.container.get('Service')
+    await this.service.start()
+
+    this.daoClaims = this.container.get('DAOClaims')
+    await this.daoClaims.start()
 
     this.logger.info('StorageWriter Started')
   }
 
   async stop() {
+    this.logger.info('Stopping StorageWriter Service')
+    await this.service.stop()
+
     this.logger.info('Stopping StorageWriter...')
     await this.router.stop()
-    this.logger.info('Stopping StorageWriter Database...')
+
+    this.logger.info('Stopping StorageWriter DAOClaims...')
```

Comment:
Why the change from `Stopping StorageWriter Database` to `Stopping StorageWriter DAOClaims`?

Reply:
Fixed, thx. I don't remember, either find/replace or I was thinking I was making mongo connection in the DAOClaims.